examples/dt/uni04delta/edpm-pre-ceph.md

```diff
@@ -0,0 +1,58 @@
+# Configuring and deploying Ceph nodes
+
+## Assumptions
+
+- The [control plane](control-plane.md) has been created and successfully
+  deployed.
+
+## Steps
+
+```bash
+# 1. Switch to openstack namespace
+
+    $ oc project openstack
+
+# 2. Change the working directory to uni04delta/edpm-pre-ceph/nodeset
+
+    $ pushd architecture/examples/dt/uni04delta/edpm-pre-ceph/nodeset
+
+# 3. Modify [values.yaml](values.yaml) file to suit your environment.
+
+    $ vi values.yaml
+
+# 4. Generate the Ceph nodeset deployment plan.
+
+    $ kustomize build --load-restrictor LoadRestrictionsNone > nodeset-pre-ceph.yaml
```

Comment:
Please remove `--load-restrictor LoadRestrictionsNone`. It's not necessary in order to build the `nodeset-pre-ceph.yaml` file.

```
[fultonj@runcible nodeset{uniDelta}]$ kustomize build > nodeset-pre-ceph-2.yaml 
[fultonj@runcible nodeset{uniDelta}]$ kustomize build --load-restrictor LoadRestrictionsNone > nodeset-pre-ceph-2.yaml 
[fultonj@runcible nodeset{uniDelta}]$ diff nodeset-pre-ceph-2.yaml nodeset-pre-ceph-1.yaml
[fultonj@runcible nodeset{uniDelta}]$ 
```

Reply:
Done